ui/00_message/ui/ArisenCard.xml

```diff
@@ -18,6 +18,7 @@
   <message>
     <key>ARISEN_CARD_TEXT_CARDMENU</key>
     <original>プロフィールの編集</original>
+	<translation>Edit Profile</translation>
```

Comment:
This entire file is likely a duplicate of ArisenCard_win.xml and is probably unused on PC. I'd suggest removing it from your commits entirely, and I can add a replication config for it on my end to mirror translations between ArisenCard.xml and ArisenCard_win.xml.

You also have numerous differences from the translations in ArisenCard_win.xml that I would consider conflicts.